content/applications/sales/crm/performance/forecast_report.rst

```diff
@@ -0,0 +1,112 @@
+===============
+Forecast report
+===============
+
+.. |caret| replace:: :icon:`fa-caret-down` :guilabel:`(down)` icon
+.. |pivot| replace:: :icon:`oi-view-pivot` :guilabel:`(pivot)` icon
+.. |list| replace:: :icon:`oi-view-list` :guilabel:`(list)` icon
+
+The *Forecast* report in the *CRM* app allows users to view upcoming opportunities and build a
+forecast of potential sales. Opportunities are grouped by the month of their expected closing date,
+and can be dragged-and-dropped to adjust the deadline.
+
+To access the *Forecast* report, navigate to :menuselection:`CRM app --> Reporting --> Forecast`.
+
+.. image:: forecast_report/reporting-menu.png
+   :align: center
+   :alt: The location of the Forecast report in the CRM app.
+
```

Comment:
In my opinion we could do without this image, just to reduce the number of overall images :)
```suggestion
```